calico-vpp-agent/routing/routing_server.go

```diff
@@ -49,14 +49,28 @@ type Server struct {
 	localAddressMap map[string]localAddress
 	ShouldStop      bool
 
-	BGPConf   *calicov3.BGPConfigurationSpec
-	BGPServer *bgpserver.BgpServer
+	BGPConf         *calicov3.BGPConfigurationSpec
+	BGPServer       *bgpserver.BgpServer
+	filters         map[string]*calicov3.BGPFilter
+	filtersPolicies map[string]*impExpPol
+	peers           map[string]*bgpapi.Peer
 
 	routingServerEventChan chan common.CalicoVppEvent
+	bgpFilterEventChan     chan common.CalicoVppEvent
 
 	nodeBGPSpec *common.LocalNodeSpec
 }
 
+type assignedPol struct {
+	PA  *bgpapi.PolicyAssignment
+	pol *bgpapi.Policy
+}
+
+type impExpPol struct {
+	imp *assignedPol
+	exp *assignedPol
+}
```

Comment:
Shouldn't we make this a property of the Peer object, so that we can find it in `w.peers[]` ?